pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -1867,5 +1958,16 @@ pub mod pallet {
 
 			Ok(final_fee)
 		}
+
+		fn evaluate_unlock_state(
+			config: &BoundedVec<u8, ConstU32<5>>,
+			account_stats: &SeasonInfo,
+		) -> bool {
```

Comment:
Looks like the wrong way.
`config[0] <= account_stats.minted`
to be true the account must have greater or equal numbers then the threashold.